hw/dv/verilator/cpp/dpi_memutil.cc

```diff
@@ -245,12 +245,13 @@ static void WriteSegment(const MemArea &m, uint32_t offset,
   // be caught at this function's callsite.
   SVScoped scoped(m.location.data());
 
-  // This "mini buffer" is used to transfer each write to SystemVerilog. It's
-  // not massively efficient, but doing so ensures that we pass 256 bits (32
-  // bytes) of initialised data each time. This is for simutil_set_mem (defined
-  // in prim_util_memload.svh), whose "val" argument has SystemVerilog type bit
-  // [255:0].
-  uint8_t minibuf[32];
+  // This "mini buffer" is used to transfer each write to SystemVerilog.
```

Comment:
As far as I can tell the original comment here isn't correct (we don't transfer 256 bits at a time unless it's a 256 bit memory), so took the opportunity to fix it.